### DISCUSSION TITLE
Rollout being automatically paused when promoted

### DISCUSSION THREAD

**Discussion by headyj** [argoproj/argo-rollouts#3778] [Q&A]
Hello,

We had a case with a rollout being automatically paused after being promoted. I did not define any pause nor analysis on my rollout.

I promoted the rollout using the kubectl plugin without the `--full`, but still I cannot find any reason in the logs explaining that. basically the command line just outputted `rollout '<rollout-name>' promoted`, but in the end it got stuck on pause.

Maybe I'm missing something about the rollout promotion process?

**Comment by kostis-codefresh** [comment]
Can you post your YAML of the rollout please?
